File: cupon_server/src/main/java/com/evgenie_tomer_itay/movies/entities/Category.java
package com.evgenie_tomer_itay.movies.entities;

public enum Category {
	FOOD(1), ELECTRICITY(2), RESTAURANT(3), VACATION(4), FURNITURES(5), HARDWARE(6);
	private int id;
	
	Category(final int id) {
		this.id = id;
	}

	public int getCategoryId() {
		return id;
	}

	public static Category getCategoryById(int id) {
		for (Category category : values()) {
			if (category.id == id)
				return category;
		}
		return null;
	}

}

File: login_application/src/src/components/CompanyHome/Add/Category.js
import './Category.css'

const Category = (props) =>
{
  
  const onChangeHandler = (event) =>
  {
    props.onYearChange(event.target.value);
  }

  return (
      <div className="expenses-filter">
        <div className="expenses-filter__control">
          <label className="expenses-filter__label">Category</label>
          <select className="expenses-filter__select" onChange={onChangeHandler}>
            <option value="FOOD">FOOD</option>
            <option value="ELECTRICITY">ELECTRICITY</option>
            <option value="RESTAURANT">RESTAURANT</option>
            <option value="VACATION">VACATION</option>
            <option value="FURNITURES">FURNITURES</option>
            <option value="HARDWARE">HARDWARE</option>

          </select>
        </div>
      </div>
  );

}

export default Category